packages/story-editor/src/components/errorBoundary/errorActions.js

```diff
@@ -86,6 +91,7 @@ function ErrorActions({ error, errorInfo }) {
         >
           {__('Reload', 'web-stories')}
         </Button>
+        {withStoryData && <CopyStoryDataToClipboard />}
```

Comment:
Do we actually need this check?

`CopyStoryDataToClipboard` can just `return null` if there's no story data available.

Reply:
This is a contrived example but if something doesn't catch up (in this case it does) ... I think it will just error out like this. 

<img width="600" alt="Screen Shot 2022-10-31 at 7 40 04 AM" src="https://user-images.githubusercontent.com/62242/198999886-7cbc0326-3f6b-4584-997c-65a600cb17cf.png">

So was thinking better to be specific vs potentially have it error out trying to get data when the Story Provider isn't wrapped out some other reason.

Reply:
My concern is that this is error prone when code gets moved around, and that we can just make `CopyStoryDataToClipboard` more robust instead of adding these`withStoryData` checks everywhere.

Something like:

```js
  // Get story data if it's available, otherwise return early.
  const { pages, current, selection, story } = useStory(({ internal }) => ({
    pages: internal?.reducerState?.pages,
    current: internal?.reducerState?.current,
    selection: internal?.reducerState?.selection,
    story: internal?.reducerState?.story,
  }));

  if (!pages || !current || !selection || !story) {
    return null;
  }
```

WDYT?

Reply:
Okay updated --- I don't think the example I showed can happen give the way the boundaries are setup,